src/app/pickup/page.tsx

```diff
@@ -31,25 +33,67 @@ import {
   PickupTimeButton,
 } from './styles';
 
+function DateInfoComponent(date: { date: unknown }) {
```

Comment:
I think you Monique and Charlotte have similar function! if you don't mind can you put this in a global folder and call it